Form-Controls/styles.css

```diff
@@ -0,0 +1,41 @@
+:root{
+    --blue:#54ADB6;
+    --black:#000000;
+    --brown:#641E16;
+    --yellow:#CC99FF;
+
+}
+
+body{color: var(--brown);
+font-family:'Raleway',sans-serif;
+font-size: 20px;}
+
+h1{text-align: center;
+color: var(--brown);}
+
+main{display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
```

Comment:
Could you combine these two rules into another one that handles both conditions?

https://developer.mozilla.org/en-US/docs/Web/CSS/place-content